app/helpers/office_helper.rb

```diff
@@ -1,4 +1,6 @@
 module OfficeHelper
+  SCHEME = 'https://'.freeze
```

Comment:
had no idea this bit is named url scheme :sweat_smile:

Reply:
Just discovered as well, well at least is what the `URI` library calls it 🤷‍♂️

Reply:
<img width="622" alt="screen shot 2019-02-07 at 17 11 27" src="https://user-images.githubusercontent.com/3481059/52429210-80f1d100-2afb-11e9-9c8a-f26c525de5ed.png">

Reply:
Yep, seemed unnecessary as I usually hear it called simply 'protocol'. But thinking outside of the web:

It simply tells you how to interpret the part of the URL after the colon.

For example, in `file://usr/share/doc`, the file tells me the part after the colon should be interpreted as a locally-available filesystem path. **This isn't identical to a protocol, because there is no transport layer or encoding - a client just uses regular local system calls to access it.**

Conversely, https://programmers.stackexchange.com specifies a protocol (https), but still requires the client to make its own choices about the physical transport used to reach it.

Taken from: https://softwareengineering.stackexchange.com/questions/294588/what-is-a-uri-scheme

Reply:
Probably not a big deal, I'd personally use `http` as default and not `https` because some websites might not be configured for ssl (I know it's 2019 but...). Not 100% sure, so not a blocker.